stakingModel/test/SimpleDAIStaking.test.js

```diff
@@ -772,6 +766,128 @@ contract("SimpleDAIStaking - staking with DAI mocks", ([founder, staker]) => {
     expect(isPaused).to.be.true;
   });
 
+  it("should not transfer excessive cdai funds when total staked is not 0 and execute recover", async () => {
+    await dai.mint(founder, web3.utils.toWei("100", "ether"));
+    await dai.approve(cDAI.address, web3.utils.toWei("100", "ether"));
+    await cDAI.mint(web3.utils.toWei("100", "ether"));
+    const cdaiBalanceFounder = await cDAI.balanceOf(founder);
+    await cDAI.transfer(simpleStaking.address, cdaiBalanceFounder);
+    let avatarBalanceBefore = await cDAI.balanceOf(avatar.address);
+    let encodedCall = web3.eth.abi.encodeFunctionCall(
+      {
+        name: "recover",
+        type: "function",
+        inputs: [
+          {
+            type: "address",
+            name: "_token"
+          }
+        ]
+      },
+      [cDAI.address]
+    );
+    await controller.genericCall(simpleStaking.address, encodedCall, avatar.address, 0);
+    let avatarBalanceAfter = await cDAI.balanceOf(avatar.address);
+    expect(avatarBalanceAfter.sub(avatarBalanceBefore).toString()).to.be.equal(
+      web3.utils.toWei("0", "ether")
+    );
+  });
+
+  it("should not transfer excessive cdai funds when total staked is 0 and not paused and execute recover", async () => {
```

Comment:
what is the difference with previous test.
if you want to cover different options do a boolean table and make sure the test description matches what you test and compare you different tests descriptions